### PR TITLE
Update readme and add a few documents

### PR DESCRIPTION
**Pull request by kicferk1**
### Context
Our readme and docs were a bit out of date.

### Changes proposed in this pull request
Write a few documentation files.

### Guidance to review
Read those files, check if they make sense to you - cause you will be using them later.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -2,6 +2,7 @@
 
 # Engage and Learn
 
+
```
Comment (callumacrae) [inline README.md:5]: ❓

File: README.md [hunk 2]
```diff
@@ -10,10 +11,10 @@
 - Yarn 1.12.x
 - Docker
 
+
 ## Setting up the app in development
 
 ### Without docker
-
```
Comment (callumacrae) [inline README.md:16]: ❓

File: README.md [hunk 3]
```diff
@@ -22,7 +23,6 @@
 6. Run `./bin/webpack-dev-server` in a separate shell for faster compilation of assets
 
 ### With docker
-
```
Comment (callumacrae) [inline README.md:25]: ❓

Comment (kicferk1) [reply README.md:25]: Fine, I will bring back whitespace

**Review by callumacrae — APPROVED** [review]

**Pull request merged**